addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -206,13 +207,15 @@ def _cancel_invoice_edi(self, invoices):
             if not response.get("error"):
                 json_dump = json.dumps(response.get("data", {}))
                 json_name = "%s_cancel_einvoice.json" % (invoice.name.replace("/", "_"))
-                attachment = self.env["ir.attachment"].create({
-                    "name": json_name,
-                    "raw": json_dump.encode(),
-                    "res_model": "account.move",
-                    "res_id": invoice.id,
-                    "mimetype": "application/json",
-                })
+                attachment = False
+                if response:
```

Comment:
```suggestion
                if json_dump:
``` ?

Reply:
Hello @jco-odoo,  No, we don't get the original request back.